source/shared/cpp/ObjectModel/pch.h

```diff
@@ -4,10 +4,8 @@
 #include "winPch.h"
 #endif
 
-#ifndef AdaptiveSharedNamespaceStart
-#define AdaptiveSharedNamespaceStart namespace AdaptiveCards {
```

Comment:
The problem this was solving was that the namespace on windows is nested and has more closing braces. We need a fix that allows us to copy this to windows without it breaking, which is why we had the braces defined here.

On windows we have:

#define AdaptiveSharedNamespaceStart namespace Windows { namespace Internal { namespace AdaptiveCards { 
#define AdaptiveSharedNamespace Windows::Internal::AdaptiveCards
#define AdaptiveSharedNamespaceEnd }}}

Reply:
When this comes down to Windows, we just need to remove `AdaptiveSharedNamespaceStart` and `AdaptiveNamespaceEnd` and then change to build using c++17, which allows nested namespace declarations (e.g. `namespace Foo::Bar::Baz { class Bleh; }`)